src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -518,12 +518,20 @@ private void AddMediaInfoTokens(Dictionary<string, Func<TokenMatch, string>> tok
                 mediaInfoSubtitleLanguages = string.Format("[{0}]", mediaInfoSubtitleLanguages);
             }
 
-            tokenHandlers["{MediaInfo Video}"] = m => mediaInfoVideo;
-            tokenHandlers["{MediaInfo Audio}"] = m => mediaInfoAudio;
+            var videoBitDepth = episodeFile.MediaInfo.VideoBitDepth.HasValue ? $"{episodeFile.MediaInfo.VideoBitDepth} Bit" : string.Empty;
+            var audioChannels = episodeFile.MediaInfo.AudioChannelPositions.IsNotNullOrWhiteSpace() ? episodeFile.MediaInfo.AudioChannelPositions.Split('/').Sum(s => decimal.Parse(s)).ToString() : string.Empty;
 
-            tokenHandlers["{MediaInfo Simple}"] = m => string.Format("{0} {1}", mediaInfoVideo, mediaInfoAudio);
+            tokenHandlers["{MediaInfo Video}"] = m => videoCodec;
```

Comment:
Should we have a migration to convert the current name to `MediaInfo Video` to `MediaInfo VideoBitDepth`? Might not be worth the effort and could drop it in v3.